readme.md

```diff
@@ -251,9 +251,7 @@ JSON body. If the `Content-Type` header is not set, it will be set to `applicati
 
 Type: `object`
 
-User data. In contrast to other options, `context` is not enumerable.
-
-**Note:** The object is never merged, it's just passed through. Got will not modify the object in any way.
+User data. `context` is shallow merged.
```

Comment:
It's worth mentioning that it **is** enumerable and if it has defined non-enumerable options they will **NOT** be merged.

#### Avoid

```
const context = {};

Object.defineProperty(context, 'secret', {
	value: 'foobar',
	enumerable: false
});
```

#### Recommended

```
const context = {
	mySecrets: {}
};

Object.defineProperty(context.mySecrets, 'secret', {
	value: 'foobar',
	enumerable: false
});
```

/cc @sindresorhus